src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -1091,12 +1091,12 @@ public String Replace(String oldValue, String newValue)
 
         public unsafe String[] Split(char separator, StringSplitOptions options = StringSplitOptions.None)
```

Comment:
`unsafe` can now be removed here and below, right?

Reply:
Yup